### PR TITLE
add web_java_in_access_log

### PR DESCRIPTION
**Pull request by frack113**
Try to add a generic Java payload detection rule in web access log.

### CODE REVIEW

File: rules/web/web_java_in_access_log.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+logsource:
+    category: webserver
+detection:
+    keywords:
+      - '%24%7B%28%23a%3D%40'
+      - '${(#a=@'
+      - '%24%7B%40java'
+      - '${@java'
+      - '\u0022java'
+      - '"java'
```
Comment (Neo23x0) [inline rules/web/web_java_in_access_log.yml:21]: Don't you think this one is prone to FPs?

Comment (frack113) [reply rules/web/web_java_in_access_log.yml:21]: Replace '\u0022java' and '"java'  by `'java.lang.'` ?
In the writeups find java.lang.ProcessBuilder and  java.lang.Runtime

Comment (Neo23x0) [reply rules/web/web_java_in_access_log.yml:21]: I'd say, only the `"java` string could be problematic

File: rules/web/web_java_in_access_log.yml [hunk 2]
```diff
@@ -17,8 +17,7 @@ detection:
       - '${(#a=@'
       - '%24%7B%40java'
       - '${@java'
-      - '\u0022java'
-      - '"java'
+      - 'java.lang.'
```
Comment (Neo23x0) [inline rules/web/web_java_in_access_log.yml:20]: What happened to `\u0022java` ?

Comment (frack113) [reply rules/web/web_java_in_access_log.yml]: Fix, the others must be significant of a java payload

**Pull request merged**